website/source/docs/install/upgrade-to-0.5.html.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: "install"
+page_title: "Upgrading to Vault 0.5"
+sidebar_current: "docs-install-upgrade-to-0.5"
+description: |-
+  Learn how to upgrade to Vault 0.5
+---
+
+# Overview
+
+This page contains the breaking changes for Vault 0.5. Please read it carefully.
```

Comment:
It'd be nice to add a note here at a very high level of what is changing and what could possibly cause breaking issues – a summary of sorts. Basically:

```
- `ttl` responses now return remaining seconds rather than value set at creation. This value now changes, and was previously static
- `nonce` token is now used in rekey operation – rekeys will fail without it
...
```